README.md

```diff
@@ -1,6 +1,16 @@
 # SaveMii
 NNID Archiver Service (source code)
 ## How do I setup this?
-Instructions will come at a later date
+This is a standard Django app,
```

Comment:
```suggestion
This is a standard Django app. You probably should know how to setup one. But if you really need steps, here you go (for development setup only):
```
some changes + made everything in a single line, cleaner

also i'll write production steps later, because if someone followed what you wrote for a production setup, he would end up with a pretty damn terrible prod setup, since the db is SQLite3, and some settings would be missing.

Reply:
yeah would be dumb if its a dev build forever